Sources/Models/Message+Subtype.swift

```diff
@@ -1,6 +1,6 @@
 
 public extension Message {
-    public enum Subtype: String {
+    enum Subtype: String {
```

Comment:
did `public` need to be dropped ?

Reply:
Resolved a warning about redundant public modifier on a public extension. I can add back if wanted?

Reply:
oh, so its implicitly `public` now.. cool!
would you mind dropping `public` from the extension and putting it on `Subtype` instead ?

I've been using that pattern lately and I feel like its a bit nicer to be explicit

Reply:
Not at all! Will update soon!